From 22c7e11df390720a12992dd85cf5ab0bf874f962 Mon Sep 17 00:00:00 2001
From: Sungbae Yoo <sungbae.yoo@samsung.com>
Date: Thu, 23 Jul 2015 14:17:03 +0900
Subject: [PATCH] exit-status: add a missing string for
 EXIT_SMACK_PROCESS_LABEL

(cherry picked from commit 94b573bd1ab54c4880ea278cb4ca08719aa4b97c)
---
 src/shared/exit-status.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/shared/exit-status.c b/src/shared/exit-status.c
index 90c83a47a8..ed1688ccd6 100644
--- a/src/shared/exit-status.c
+++ b/src/shared/exit-status.c
@@ -151,6 +151,9 @@ const char* exit_status_to_string(ExitStatus status, ExitStatusLevel level) {
 
                 case EXIT_BUS_ENDPOINT:
                         return "BUS_ENDPOINT";
+
+                case EXIT_SMACK_PROCESS_LABEL:
+                        return "SMACK_PROCESS_LABEL";
                 }
         }
 
